src/apis/abusiveexperiencereport/v1.ts

```diff
@@ -40,6 +41,17 @@ export namespace abusiveexperiencereport_v1 {
   }
 
   interface StandardParameters {
+    /**
+     * Auth client or API Key for the request
+     */
+    auth?:
```

Comment:
nit, but still: the keys were sorted here, maybe `auth` can go below `alt`? :)

Reply:
All of those standard parameters are actually generated from the discovery file.  This would be silly hard :/